src/dom.ts

```diff
@@ -20,6 +20,50 @@ for (const param in tagWrap) {
 	tw.post = '</' + tw.reverse().join('></') + '>';
 }
 
+function validateClass(token: string): void {
+	if (!token || token === '') {
+		throw new Error('An invalid or illegal string was specified');
+	}
+	if (/\s/.test(token)) {
+		throw new Error('String contains an invalid character');
+	}
+}
+
+/**
+ * Adds one or more CSS class names to an Element without duplication
+ *
+ * @param element The Element to which to add CSS classes
+ * @param classes An array of string CSS classes to add to the Element
```

Comment:
I wouldn't say "an array" here since it's not actually an array, it's a rest argument.  I'd probably just say "one or more CSS class strings".